src/trumbowyg.js

```diff
@@ -1684,8 +1703,8 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
                     $.each(fields, function (fieldName, field) {
                         var n = field.name || fieldName;
 
-                        var $field = $('input[name="' + n + '"]', $form),
-                            inputType = $field.attr('type');
+                        var $field = $(':input[name="' + n + '"]', $form),
+                            inputType = $field[0].nodeName;
```

Comment:
This will return input, textarea, select, but will not return checkbox nor radio, am I right?

Reply:
i just did up a quick jsfiddle
https://jsfiddle.net/46Le9quy/
it looks like it does support most fields

Reply:
OK... just select is `select-one` or `select-multiple` when adding attribute `multiple`, idk if its intuitive

Reply:
The difference between the code here and your demo is `.type` vs `.nodeName`, I will fix that :)